.rubocop.yml

```diff
@@ -49,6 +57,11 @@ Metrics/LineLength:
   # To make it possible to copy or click on URIs in the code, we allow lines
   # containing a URI to be longer than Max.
   AllowURI: true
+  Exclude:
+    - db/migrate/20161011184702_devise_create_admin_users.rb
+    - db/migrate/20161011151353_devise_create_users.rb
+    - db/migrate/20161027190856_create_delayed_jobs.rb
+    - db/migrate/20181102142200_create_active_storage_tables.active_storage.rb
```

Comment:
Shouldn't we exclude all future migrations from this cop?

Reply:
I consider migrations as part of the app code and think we should care about it's quality as well. If we exclude every migration file we'd be decreasing quality there. That said, I don't have a strong opinion on any option, not sure what the rest of the people think, should we:
1. Fix those migrations
2. Exclude those migrations (like it's now)
3. Exclude every migration
?